File: src/main/java/com/hyecheon/antteshop/entity/UserFileInfo.java
package com.hyecheon.antteshop.entity;

import lombok.*;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

/**
 * User: hyecheon lee
 * Email: rainbow880616@gmail.com
 * Date: 2021/08/21
 */
@EqualsAndHashCode(callSuper = true)
@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class UserFileInfo extends BaseWithUserEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String originalFileName;

    private String type;

    private String path;

    private String size;
}

File: src/main/kotlin/com/hyecheon/antteshop/domains/entity/UserFileInfo.kt
package com.hyecheon.antteshop.entity

import lombok.*
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id

/**
 * User: hyecheon lee
 * Email: rainbow880616@gmail.com
 * Date: 2021/08/21
 */
@Entity
@Builder
class UserFileInfo(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null,
    var originalFileName: String? = null,
    var type: String? = null,
    var path: String? = null,
    var size: String? = null,
) : BaseWithUserEntity()